README.md

```diff
@@ -80,20 +80,6 @@ project with `cargo` and build it. Follow along below:
 
```

Comment:
Looks like you forgot to remove the *Follow along below:* part above.

Reply:
Woops, thanks